lib/std/build.zig

```diff
@@ -1626,6 +1627,17 @@ pub const LibExeObjStep = struct {
         other_step: *LibExeObjStep,
     };
 
+    const IncludePathVisibility = packed struct {
```

Comment:
s/IncludePathVisibility/IncludePathVisible ?

Reply:
changed to IncludePathAccess